### PR TITLE
Postgres Preset: Add support for timezone

### PR DESCRIPTION
**Pull request by pawndev**
Hey !

I needed something like this for a project I'm working on, and though it would be useful for someone else.
What do you think about this ?

It was fun doing this, if you need anything else, let me know !

P.S. Big thanks for this project !

### CODE REVIEW

**Comment by orlangure** [comment]
Hi! Thank you for contribution, it looks great and seems useful.

I'll take a closer look in a week when I'm back from vacation.

**Comment by pawndev** [comment]
Okey ! Have a nice vacation ! 🙂

File: preset/postgres/preset_test.go [hunk 1]
```diff
@@ -71,11 +78,18 @@ func TestPreset_withDefaults(t *testing.T) {
 	connStr := fmt.Sprintf(
 		"host=%s port=%d user=%s password=%s  dbname=%s sslmode=disable",
 		container.Host, container.DefaultPort(),
-		"gnomock", "gnomick", "mydb",
+		"postgres", "password", "postgres",
```
Comment (orlangure) [inline preset/postgres/preset_test.go:81]: Nice, I forgot why it was something else before😼
Now I don't understand why it worked...

File: preset/postgres/options.go [hunk 2]
```diff
@@ -43,3 +43,10 @@ func WithVersion(version string) Option {
 		o.Version = version
 	}
 }
+
+// WithTimezone set the postgres timezone.
```
Comment (orlangure) [inline preset/postgres/options.go:47]:
```suggestion
// WithTimezone sets the timezone in this container.
```

**Review by orlangure — APPROVED** [review]
Great job, thank you for the contribution 😻

**Comment by pawndev** [comment]
My pleasure ! I will try to find some other task to contribute to gnomock 😃

**Pull request merged**